tests/components/brother/test_diagnostics.py

```diff
@@ -0,0 +1,86 @@
+"""Test Brother diagnostics."""
+from datetime import datetime
+import json
+from unittest.mock import Mock, patch
+
+from homeassistant.util.dt import UTC
+
+from tests.common import load_fixture
+from tests.components.brother import init_integration
+from tests.components.diagnostics import get_diagnostics_for_config_entry
+
+
+async def test_entry_diagnostics(hass, hass_client):
+    """Test config entry diagnostics."""
+    entry = await init_integration(hass, skip_setup=True)
+
+    test_time = datetime(2019, 11, 11, 9, 10, 32, tzinfo=UTC)
+    with patch("brother.datetime", utcnow=Mock(return_value=test_time)), patch(
+        "brother.Brother._get_data",
+        return_value=json.loads(load_fixture("printer_data.json", "brother")),
+    ):
+        await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
+
+    result = await get_diagnostics_for_config_entry(hass, hass_client, entry)
+
+    result["info"].pop("entry_id")
+
+    assert result["info"] == {
+        "data": {"host": "localhost", "type": "laser"},
+        "disabled_by": None,
+        "domain": "brother",
+        "options": {},
+        "pref_disable_new_entities": False,
+        "pref_disable_polling": False,
+        "source": "user",
+        "title": "HL-L2340DW 0123456789",
+        "unique_id": "0123456789",
+        "version": 1,
+    }
+
+    assert result["data"] == {
```

Comment:
Shouldn't you be able to use `load_fixture` here ?

Reply:
Yes, I can move data to json file and use it as fixture.